Header header
string robot_name
int8 robot_id
bool is_ready
bool is_arrived
float32 x
float32 y
float32 theta
